### PR TITLE
localize data_center.md

### PR DESCRIPTION
**Pull request by Submarinee**
localize data_center.md

### CODE REVIEW

File: content/zh/data_center.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: 数据中心
+status: Feedback Appreciated
+category:技术
```
Comment (hanyuancheung) [inline content/zh/data_center.md:4]:
```suggestion
category: 技术
```
unless the pipeline will not pass.

**Review by hanyuancheung — APPROVED** [review]
lgtm

File: content/zh/data_center.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: 数据中心
+status: Feedback Appreciated
+category: 技术
+---
+
+## 是什么
+
+数据中心是专门设计用于容纳计算机（通常是服务器）的专用建筑物或设施。 数据中心往往连接到高速互联网线路，尤其是在主打 [云计算](/cloud_computing/) 的数据中心。 数据中心所在的建筑物还配备了即使在发生负面事件时也能维持服务的设备，例如在停电期间提供电力的发电机，以及处理计算机产生的废热的强大空调。
```
Comment (Rocksnake) [inline content/zh/data_center.md:9]:
```suggestion
数据中心是专门设计用于容纳计算机（通常是服务器）的专用建筑物或设施。 数据中心往往连接到高速互联网线路，尤其是专注于 [云计算](/cloud_computing/) 的数据中心。 数据中心所在的建筑物还配备了即使在发生负面事件时也能维持服务的设备，例如在停电期间提供电力的发电机，以及处理计算机产生的废热的强大空调。
```

**Review by Rocksnake — APPROVED** [review]
lgtm

**Pull request merged**